mpqp/execution/devices.py

```diff
@@ -330,3 +330,36 @@ def is_processor(self) -> bool:
             True if the device is a processor, False otherwise.
         """
         return self.name.startswith("PROCESSOR")
+
+
+class AZUREDevice(AvailableDevice):
+    """Enum regrouping all available devices provided by AZURE."""
+
+    IONQ_SIMULATOR = "ionq.simulator"
+    IONQ_QPU = "ionq.qpu"
+    IONQ_QPU_ARIA_1 = "ionq.qpu.aria-1"
+    IONQ_QPU_ARIA_2 = "ionq.qpu.aria-2"
+
+    QUANTUM_SIM_H1_1 = "quantinuum.qpu.h1-1"
+    QUANTUM_SIM_H1_1SC = "quantinuum.sim.h1-1sc"
+    QUANTUM_SIM_H1_1E = "quantinuum.sim.h1-1e"
+
+    RIGETTI_SIM_QVM = "rigetti.sim.qvm"
+    RIGETTI_SIM_QPU_ANKAA_2 = "rigetti.qpu.ankaa-2"
+
+    MICROSOFT_ESTIMATOR = "microsoft.estimator"
+
+    def is_remote(self):
+        return True
+
+    def is_gate_based(self) -> bool:
+        return True
+
+    def is_simulator(self) -> bool:
+        return True
+
+    def is_noisy_simulator(self) -> bool:
```

Comment:
none of those support noise ?

Reply:
for now yes. we will fix it with noise for cirq